### PR TITLE
[Mobile Design System] Add missing USWDS colors required for mobile

### PR DESCRIPTION
**Pull request by narin**
## Chromatic

https://va-mobile-library&#x2F;7425-add-uswds-colors--60f9b557105290003b387cd5.chromatic.com
## Description
Closes https://github.com/department-of-veterans-affairs/va-mobile-app/issues/7425

Adds some missing USWDS colors that are required for the VA Mobile component library:
- gray-warm-90
- blue-vivid-80
- blue-vivid-30
- blue-vivid-20
- blue-vivid-10
- blue-vivid-5
- red-vivid-80
- red-vivid-40
- green-cool-vivid-80
- yellow-vivid-80

After adding these colors to `tokensToKeep` in `copy-uswds-color-tokens.js`, re-ran `yarn build` to regenerate `dist/` files and `uswds.json`.

## QA Checklist
- [ ] Component maintains 1:1 parity with design mocks
- [ ] Text is consistent with what's been provided in the mocks
- [ ] Component behaves as expected across breakpoints
- [ ] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
- [ ] Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [ ] Tab order and focus state work as expected
- [ ] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
- [ ] New components are covered by e2e tests; updates to existing components are covered by existing test suite
- [ ] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)

## Acceptance criteria
- [ ] QA checklist has been completed
- [ ] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by jamigibbs — APPROVED** [review]

**Review by micahchiang — CHANGES_REQUESTED** [review]
temporarily requesting changes here until semantic naming is resolved. CC @humancompanion-usds and @jamigibbs

File: packages/css-library/tokens/uswds.json
Comment (micahchiang) [inline packages/css-library/tokens/uswds.json:1]: Hey @narin! These additions look fine to me, however we also need the mobile team to provide semantic names representative of what these tokens do. All VADS color variables are ultimately generated out of [this color.json file](https://github.com/department-of-veterans-affairs/component-library/blob/2060dceff8261812c411f5b102212da9378bbfa1/packages/css-library/tokens/color.json#L1) and use uswds tokens as the values for semantically named variable names (for example: `color.primary.dark` eventually generates a css variable called `--color-primary-darker`.) 

In an effort to maintain parity between the mobile app and the web platform, we need the mobile team to include entries in `color.json` with semantic names. These would get generated in the `variables.json` file the mobile team uses as something like: 

```json
    "darker": {
      "value": "#162e51",
      "filePath": "tokens/color.json",
      "isSource": true,
      "original": {
        "value": "{uswds-system-color-blue-warm-vivid-80}"
      },
      "name": "color-primary-darker",
      "attributes": {
        "category": "color",
        "type": "primary",
        "item": "darker"
      },
      "path": [
        "color",
        "primary",
        "darker"
      ]
    },
```

where you'd then be able to use `color-primary-darker` directly, maintaining consistency with VA.gov's web platform.

Comment (micahchiang) [reply packages/css-library/tokens/uswds.json:1]: @humancompanion-usds addresses some of this in this issue - https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2085

**Comment by narin** [comment]
Closing for now until we've reconciled the corresponding semantic tokens and https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2085 as discussed in this Slack thread: https://dsva.slack.com/archives/C06D8NB8R8V/p1705686281448499

**Pull request closed without merging**